sdks/python/setup.cfg

```diff
@@ -25,3 +25,30 @@ verbosity=2
 # fast_coders module which is not available when running unit tests:
 # fast_coders_test and typecoders_test.
 exclude=fast_coders_test|typecoders_test
+
+# Configuration for code coverage.
+# .coveragerc to control coverage.py
+[coverage:run]
+branch = True
+source = apache_beam
+
+[coverage:report]
+# Regexes for lines to exclude from consideration
+exclude_lines =
```

Comment:
Maybe exclude the auto-generated files in portability/api?

Reply:
According to the comment in files, I guess we can exclude all `*_pb2.py` and `pb2_grpc.py`?